infra/utils/e2e-sw-validator.js

```diff
@@ -0,0 +1,16 @@
+const fakeRunAndCompare = require('./validator/fake-run-and-compare');
```

Comment:
I'd kill this test for now.

It caused a great deal of headaches and needs to be re-considered.

Reply:
I'd like to maintain parity with the v2 test suite in this PR. I know the test is problematic and the tests may change in the future, but I ended up finding some broken things that I wouldn't have otherwise noticed during the move by virtue of ensuring the existing tests worked.